cluster/juju/layers/kubernetes-master/reactive/kubernetes_master.py

```diff
@@ -936,10 +936,12 @@ def addons_ready():
 def loadbalancer_kubeconfig(loadbalancer, ca, client):
```

Comment:
Why is this method called both by the reactive framework and by our code directly?